src/Routing/Router.php

```diff
@@ -978,13 +1005,82 @@ public static function scope($path, $params = [], $callback = null)
         $options = [];
         if (is_array($params)) {
             $options = $params;
-            unset($params['routeClass'], $params['extensions']);
+            $cache = (string)Hash::get($params, '_cache');
```

Comment:
It would be more readable and potentially faster to do `$cache = $params['_cache'] ?? null`